.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
Can we checkout the specific tag instead of master?

Reply:
I don't think so.. Build should pick the latest code from `main` branch only. This will make sure the `main` branch is always at working state. The expectation is User to create a Tag from `master/main` branch only

Reply:
I checked the doc, it seems the `actions/checkout@v2` always checkouts the reference which caused the event.
 

> # The branch, tag or SHA to checkout. When checking out the repository that
>  # triggered a workflow, this defaults to the reference or SHA for that event.
>  # Otherwise, uses the default branch.
>     ref: ''

Reply:
@arajkumar Ya, Doc. says
 
> Otherwise, uses the default branch.

AFAIK, if `ref` is not set, it will pick from default branch and our repo has `main` as the default branch